vulnerabilities/models.py

```diff
@@ -1369,6 +1370,11 @@ def save(self, *args, **kwargs):
         self.unique_content_id = checksum.hexdigest()
         super().save(*args, **kwargs)
 
+    def save(self, *args, **kwargs):
+        advisory_data = self.to_advisory_data()
+        self.unique_content_id = compute_content_id(advisory_data, include_metadata=False)[:31]
```

Comment:
Why trim the hash to 32 characters when we're increasing the max_length for `unique_content_id` to 64?

```suggestion
        self.unique_content_id = compute_content_id(advisory_data, include_metadata=False)
```